sbin/build.sh

```diff
@@ -442,16 +442,28 @@ getGradleHome() {
   echo $gradleJavaHome
 }
 
+getGradleUserHome() {
+  local gradleUserHome=""
+
+  if [ -z "${BUILD_CONFIG[GRADLE_USER_HOME]}" ]; then
+    gradleUserHome="${BUILD_CONFIG[WORKSPACE_DIR]}/.gradle"
```

Comment:
Maybe flip this logic around so we're checking for the positive as opposed to the negative?  Not sure how easy that would be.